demos/k8s-demo/README.md

```diff
@@ -1,303 +1,142 @@
-    protocol: pg
-    address: localhost:5432
+**Important: The application developer never needs any credentials to connect
+to the database.**
 
-handlers:
-  - name: pets-pg-handler
-    listener: pets-pg-listener
-    credentials:
-      - name: address
-        provider: kubernetes
-        id: quick-start-backend-credentials#address
-      - name: username
-        provider: kubernetes
-        id: quick-start-backend-credentials#username
-      - name: password
-        provider: kubernetes
-        id: quick-start-backend-credentials#password
-```
-Note: we don't specify an `sslmode` in the Secretless Broker config, so it will
-use the default `require` value.
+In particular, as the application developer you do not know any of the secrets
+in **admin_config.sh**.
 
-### Steps for the non-privileged user (i.e. developer)
+**As the application developer:**                                                                                                                                       
 
-**Note:** None of these steps require the information in `admin_config.sh` - the person deploying the application needs to know _nothing_ about the secret values required to connect to the PostgreSQL database!!
+1. Configure the application to connect to PostgreSQL via Secretless                                                                                                    
+1. Deploy the application and the Secretless sidecar                                                                                                                    
+1. Test the application
 
-#### 1. Configure application to access the database at `localhost:5432`
+To perform all these steps in one go, run:
 
-In the application manifest, we set the `DB_URL` environment variable to `localhost:5432`, so that when the application is deployed it will open the connection to the PostgreSQL backend via the Secretless Broker.
-
-#### 2. Deploy application
-
-To deploy the application with the Secretless Broker, run:
 ```bash
-./03_deploy_app.sh
-```
-```
->>--- Create and store Secretless configuration
-configmap/quick-start-application-secretless-config created
->>--- Start application
-deployment.apps/quick-start-application created
-service/quick-start-application created
-Waiting for quick-start-application to be ready
-...
-Ready!
+./02_app_developer_steps
 ```
 
-### Try it out!
-
-That's it! You've configured your application to connect to PostgreSQL via the Secretless Broker, and we can try it out to validate that it's working as expected.
-
-#### Use the pet store app
-
-`POST /pet` to add a pet - the request must include `name` in the JSON body
-```bash
-APPLICATION_URL=$(. ./admin_config.sh; echo ${APPLICATION_URL})
-
-curl \
-  -i \
-  -d '{"name": "Mr. Snuggles"}' \
-  -H "Content-Type: application/json" \
-  ${APPLICATION_URL}/pet
-```
-```bash
-HTTP/1.1 201
+<p></p>
+<details>
+  <summary>View expected output</summary>
+  <pre>
+>>--- Start application
```

Comment:
same as above, with output:
```
>>--- Start application

deployment "quick-start-application" created
service "quick-start-application" created
Waiting for application to boot up
(This may take more than 1 minute)
..........Done


>>--- Add a Sample Pet

HTTP/1.1 201 
Location: http://192.168.99.100:30002/pet/1
Content-Length: 0
Date: Thu, 14 Mar 2019 16:25:30 GMT



>>--- Retrieve All Pets

HTTP/1.1 200 
Content-Type: application/json;charset=UTF-8
Transfer-Encoding: chunked
Date: Thu, 14 Mar 2019 16:25:31 GMT

[{"id":1,"name":"Mr. Snuggles"}]

All finished!  Secretless is working!
```